sources/web/datalab/static.ts

```diff
@@ -112,6 +116,23 @@ function datalabFileExists(filePath: string) {
 }
 
 /**
+ * Sends a static 'custom.css' file located within the user's config directory.
+ *
+ * @param userId the ID of the current user.
+ * @param response the out-going response associated with the current HTTP request.
+ */
+function sendUserCustomTheme(userId: string, response: http.ServerResponse): void {
+    var customThemePath = path.join(settings.getUserConfigDir(userId), CUSTOM_THEME_FILE);
+    fs.stat(customThemePath, function(e, stats) {
```

Comment:
> In this particular case, once we have the UI ready we can do a cache refreshing on an UI event 

The UI will only be for the setting, not for editing the theme file. As such, refreshing the cache based on UI events will not be sufficient.